benchmarker.py

```diff
@@ -71,6 +84,61 @@ def get_factorio_version(factorio_bin: str, full: bool = False) -> str:
     return result
 
 
+def migrate_map(factorio_bin: str, map: str, inplace: bool, custom_script: str | None) -> None:
+    """migrate map to a new factorio version."""
+    print("migrating map:" + map)
+    if not inplace:
+        oldmap = Path(map)
+        newmap = Path(
+            os.path.splitext(map)[0] + get_factorio_version(factorio_bin).replace(".", "") + ".zip"
```

Comment:
```suggestion
            os.path.splitext(map)[0] + "_" + get_factorio_version(factorio_bin).replace(".", "_") + ".zip"
```
imho so the version will be clearer

Reply:
but longer. yes it's more readable, but the filenames are already really long most of the time. and so adding even more (especially with multiple migrations) isn't optimal.